backend/graphql/team/mutation.graphql

```diff
@@ -0,0 +1,6 @@
+extend type Mutation {
+    updateTeamDashboard(
+        id: ID! @rules(apply: ["required", "integer"])
+        dashboard_content: String! @rules(apply: ["required"])
```

Comment:
What about for the content title? 

See screenshot for reference: https://framgiaph.backlog.com/view/SUN_OVERFLOW-180

Reply:
There's no content title na daw